### ISSUE TITLE
OCSP Error & Animations

### ISSUE THREAD

**Issue by sprvoll2cx86** [pyllyukko/user.js#334]
Hello!
I love the user.js and I have been using it for a year now. Awesome job! Yesterday I intalled user.js on a fresh install of Firefox 55.0.2 but when I tried to connect to GitHub or Google for example, it shows me this error:

`An error occurred during a connection to github.com. The OCSP server experienced an internal error. Error code: SEC_ERROR_OCSP_SERVER_ERROR`

What could be causing this error? It only happens after I enable user.js.

Also how can I enable jQuery animations in user.js. For example https://alvarotrigo.com/multiScroll/ does not work, or if it works on some websites - it lags very much.

Thank you very much! :)

**Comment by pyllyukko** [comment]
> I love the user.js and I have been using it for a year now. Awesome job!

Thanks! 😊 

> Yesterday I intalled user.js on a fresh install of Firefox 55.0.2 but when I tried to connect to GitHub or Google for example, it showed me this error:
> 
> An error occurred during a connection to github.com. The OCSP server experienced an internal error. Error code: SEC_ERROR_OCSP_SERVER_ERROR
> 
> I noticed that it happens after I enable NoScript. So I needed to disable & enable NoScript after I installed user.js in order to work.

So is this 100% reproducible? Or was it just one time? As I've said [here](https://github.com/pyllyukko/user.js/issues/321#issuecomment-311464490), OCSP servers do have the tendency of failing every now and then and ```security.OCSP.require``` makes sure that you don't get to those sites when that happens.

> How can I enable jQuery animations in user.js. For example https://alvarotrigo.com/multiScroll/ works but it lags very much. Which setting is causing this?

It was quite smooth for me. Could it be because webGL is disabled and you don't get the benefit of HW rendering?

**Comment by sprvoll2cx86** [comment]
> So is this 100% reproducible? Or was it just one time? As I've said here, OCSP servers do have the tendency of failing every now and then and security.OCSP.require makes sure that you don't get to those sites when that happens.

I am not sure will it be 100% reproducible with NoScript. But while ago I found a reproducible way with uBlock Origin. So on a fresh Firefox I install user.js and then install uBlock. After that when you go to uBlock settings > My rules and apply the following rules:
```
no-large-media: * true
no-remote-fonts: * true
* * * block
* * 1p-script block
* * 3p block
* * 3p-frame block
* * 3p-script block
* * image block
* * inline-script block 
```
If these rules are applied then an OCSP server error will show up when trying to connect to Facebook or Github for example. I noticed that removing the following rules is causing the error:
```
behind-the-scene * 3p noop
behind-the-scene * 3p-frame noop
```
> It was quite smooth for me. Could it be because webGL is disabled and you don't get the benefit of HW rendering?
I just tried on a fresh Firefox with enabled WebGL user.js but the animation lagging is still there. It is happening after I install user.js.

Thank you for your support!

**Comment by savyajha** [comment]
> How can I enable jQuery animations in user.js. For example https://alvarotrigo.com/multiScroll/ works but it lags very much. Which setting is causing this?

Can confim this, and it's happening because of `privacy.resistFingerprinting`. I don't mind this pref normally, but it's causing quite a hassle. [This](https://github.com/pyllyukko/user.js/issues/333) is also being caused by the same pref. I'm quite sure that while I like the tor uplift project, it would be a better idea to have all the things `privacy.resistFingerprinting` does split among various keys that can be enabled/disabled at will.

**Comment by sprvoll2cx86** [comment]
> Can confim this, and it's happening because of privacy.resistFingerprinting. I don't mind this pref normally, but it's causing quite a hassle. This is also being caused by the same pref. I'm quite sure that while I like the tor uplift project, it would be a better idea to have all the things privacy.resistFingerprinting does split among various keys that can be enabled/disabled at will.

Yes, disabling `privacy.resistFingerprinting` fixes the problem immediately. Thank you very much!

**Comment by savyajha** [comment]
I have written a bug report over [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1394735). Hopefully we'll get a fix soon.

**Comment by pyllyukko** [comment]
I guess this is out of our hands now and all we can do is to hope Mozilla fixes this. Thanks for creating the bug report, @savyajha.
